test/client_track_test.rb

```diff
@@ -70,6 +81,17 @@ def test_track_with_long_string_property_value
         'long; 1025 is above the 1024-character limit', exception.message
   end
 
+  def test_track_with_long_symbol_identity
```

Comment:
I'm fine with this naming style if you prefer it, as long as there is `string` / `symbol` in the method name, so they don't overlap.